ClientApp/viking-vault/src/components/SideBar.tsx

```diff
@@ -34,25 +34,28 @@ class SideBar extends React.Component<any, ISideBarState> {
         return(
             <div className={"sidebar-" + this.state.show} id="mySidebar">
                 <div className="regular-div">
-                    <p className="p"> <img className="img" src={dashboard} alt=""/> My Dashboard  <img src={arrow} className = {this.state.show ? "transform-none img" : "transform-reverse img"} alt="" onClick={this.clickHandler}/></p>
+                        <img className="img" src={dashboard} alt=""/> 
+                        &nbsp;
+                        <span className = {"dashboard-" + this.state.show}>Dashboard</span>
+                        <img src={arrow} className = {this.state.show ? "transform-none img" : "transform-reverse img"} alt="" onClick={this.clickHandler}/>
```

Comment:
Small thing, but if the alt attribute is empty then it's kind of redundant to be there